apps/dot-voting/app/store/events.js

```diff
@@ -28,14 +24,8 @@ export const handleEvent = async (state, event, settings) => {
   case 'StartVote':
     nextState = await startVote(nextState, returnValues)
     break
-  case 'EntryAdded':
-    nextEntries = await onEntryAdded({ entries, addressBook }, returnValues)
-    nextState.entries = nextEntries
-    break
-  case 'EntryRemoved':
-    nextEntries = await onEntryRemoved({ entries, addressBook }, returnValues)
-    nextState.entries = nextEntries
-    break
+  case 'UpdateQuorum':
+  case 'UpdateMinimumSupport':
```

Comment:
These 2 events are not handled right now, do they involve changes in the state?

Reply:
It's just a hook for us in the future. Right now both of these will "fall through" to the `default` case below.

Reply:
I could be convinced to take it out, but I like that this gives more visibility to what events are available in the contract.

Reply:
I'm in favor of keeping them. It's like a comment-less todo